Documentation/Storage-Configuration/NFS/nfs-csi-driver.md

```diff
@@ -6,7 +6,7 @@ title: CSI provisioner and driver
     This feature is experimental and will not support upgrades to future versions.
 
 In version 1.9.1, Rook is able to deploy the experimental NFS Ceph CSI driver. This requires Ceph
-CSI version 3.7.0 or above. We recommend Ceph v16.2.7 or above.
+CSI version 3.8.0 or above. We recommend Ceph v16.2.7 or above.
```

Comment:
Let's remove this paragraph since the comment about v1.9.1 does not apply to v12. @BlaineEXE sound good?

Reply:
Yes, I think the nfs csi driver is stable and backward compatible now, this can be removed.